Makefile

```diff
@@ -71,14 +71,17 @@ lint-container: ## Lint the Symfony DI container
 lint-twig: ## Lint Twig files
 	@bin/console lint:twig templates/
 
+fix-twig: ## Fix Twig files with Twig CS Fixer
+	@vendor/bin/twig-cs-fixer lint --fix ./templates
+
 lint-yaml: ## Lint YAML files
 	@bin/console lint:yaml --parse-tags config/
 
 cs: ## Run all CS checks
 cs: fix-php stan
 
 lint: ## Run all lints
-lint: lint-php lint-container lint-twig lint-yaml
+lint: lint-php lint-container lint-twig lint-yaml fix-twig
```

Comment:
This is only lint commands (which doesn't modify files)
but the fix-twig is fixing files so it should be here. (only something like lint-twig-cs since there is already lint-twig)